contracts/ethereum/scripts/deploy.js

```diff
@@ -146,22 +134,116 @@ async function deployStableVault() {
   // await vault.setUserCompoundPreference(true)
   // FIXME add real address or way to user set its own wallet.
   // this wallet is the second hardhat wallet
-  await vault.setUserCompoundPreference(true, "0x70997970C51812dc3A010C7d01b50e0d17dc79C8")
+  await vault.setUserCompoundPreference(false, admin.address) // LOCALNETWORK: False, otherwise will fail because Spark/AAve does not exist in local network
   console.log("StableVault deployed to:", vault.address)
 
   copyVaultABI()
-  return vault.address
+  return {
+    address: vault.address,
+    deploymentBlock: deployReceipt.blockNumber,
+  }
+}
+
+async function deployKyodoRegistry(agreementData, vaultData, fakeStableAddress, kyodoTreasureContractAddress, communityTreasureContractAddress) {
+  const KyodoRegistry = await ethers.getContractFactory("KyodoRegistry");
+  const [admin] = await ethers.getSigners();
+  const kyodoRegistry = await KyodoRegistry.deploy(admin.address);
+  await kyodoRegistry.deployed()
+
+  const keysToUpdate = {
+    'FAKE_STABLE_ADDRESS': fakeStableAddress,
+    'AGREEMENT_CONTRACT_ADDRESS': agreementData["address"],
+    'VAULT_CONTRACT_ADDRESS': vaultData["address"],
+    'KYODO_TREASURE_CONTRACT_ADDRESS': kyodoTreasureContractAddress,
+    'COMMUNITY_TREASURE_CONTRACT_ADDRESS': communityTreasureContractAddress
+  };
+
+  for (const [key, value] of Object.entries(keysToUpdate)) {
+    try {
+      const tx = await kyodoRegistry.createRegistry(key, value);
+      
+      await tx.wait();
+    } catch (error) {
+      console.log(`error trying to save ${key} on KyodoRegistry`, error);
+    }
+  }
+
+  const address = await kyodoRegistry.getRegistry("AGREEMENT_CONTRACT_ADDRESS");
+  console.log("address Saved", address)
+
+  console.log("KyodoRegistry deployed to:", kyodoRegistry.address);
+  return kyodoRegistry.address;
+}
+
+async function deployMultiSig(multiSigName, signer, owners) {
```

Comment:
`multiSigName` is a dependency for console log, please don't add dependencies for console logs